### ISSUE TITLE
e2e flake: 502 Bad Gateway in kubemark startup

### ISSUE THREAD

**Issue by gmarek** [kubernetes/kubernetes#25872]
```
20:36:05 latest: digest: sha256:276c5743a7442f91337b42724ab921e9b6f6e4547d91f77fa81681ea3c15305f size: 7447
20:37:05 Received unexpected HTTP status: 502 Bad Gateway
```

This happened in Makefile which reads:
```
all:
	docker build -t gcr.io/$(PROJECT)/kubemark .
	gcloud docker push gcr.io/$(PROJECT)/kubemark
```

I guess it's some kind of infrastructure problem. @fejta please reassign.

https://storage.googleapis.com/kubernetes-jenkins/logs/kubernetes-kubemark-500-gce/3297/build-log.txt

@kubernetes/sig-testing

**Comment by fejta** [comment]
Looks like a flake of `docker build -t gcr.io/kubernetes-scale/kubemark .` however I cannot determine which line of `./test/kubemark/start-kubemark.sh` triggers this call.

**Comment by gmarek** [comment]
It's the `make` call in line https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/start-kubemark.sh#L60

**Comment by fejta** [comment]
Got it thanks! It sounds like `start-kubemark.sh` needs be robust to 5xx responses.

**Comment by gmarek** [comment]
Fixed by #26533
